tools/StaticAnalysis/Exceptions/Az.Storage/MissingAssemblies.csv

```diff
@@ -2,3 +2,4 @@
 "Az.Storage","Mono.Posix.NETStandard","1.0.0.0","Microsoft.WindowsAzure.Storage.DataMovement","0","3000","Missing assembly Mono.Posix.NETStandard referenced from Microsoft.WindowsAzure.Storage.DataMovement","Ensure that the assembly is included in the Wix file or directory"
 "Az.Storage","Microsoft.Win32.Registry","4.1.1.0","System.Management.Automation","0","3000","Missing assembly Microsoft.Win32.Registry referenced from System.Management.Automation","Ensure that the assembly is included in the Wix file or directory"
 "Az.Storage","Microsoft.PowerShell.CoreCLR.Eventing","6.1.0.0","System.Management.Automation","0","3000","Missing assembly Microsoft.PowerShell.CoreCLR.Eventing referenced from System.Management.Automation","Ensure that the assembly is included in the Wix file or directory"
+"Az.Storage","Mono.Posix.NETStandard","1.0.0.0","Microsoft.WindowsAzure.Storage.DataMovement.PowerShell","0","3000","Missing assembly Mono.Posix.NETStandard referenced from Microsoft.WindowsAzure.Storage.DataMovement.PowerShell","Ensure that the assembly is included in the Wix file or directory"
```

Comment:
DMlib reference Mono.Posix.NETStandard, because DMlib need to support Symbolic link, and in netStandard lib there are no support for Symbolic link, so we need to refer this dll to support Symbolic link. 
Actually, Powershell only use the single file transfer API from DMlib, and Single file transfer API is not related with Symbolic link (Directory transfer will use Symbolic link), so actually PSH won't call the API in it. 

We have this exception in the before release (with original DMlib name), add it just because DMlib has a new name. the original exception is added in https://github.com/Azure/azure-powershell/commit/1476c127864f118f768fc9b2b9c29f6d458e0a3a#diff-dafe08ce1ad7ca164c63c50f25f410ddR2

Reply:
Sounds good, we should add a similar exception to the existing DMLib